src/java/org/apache/cassandra/db/commitlog/CommitLogSegmentManagerCDC.java

```diff
@@ -380,14 +430,12 @@ private void addSize(long toAdd)
     @VisibleForTesting
     public long updateCDCTotalSize()
     {
+        long sleepTime = DatabaseDescriptor.getCDCDiskCheckInterval() + 10L;
```

Comment:
Discussed offline; up this to 50 and hold our noses and it should have enough buffer for really overloaded test environments, since it's ultimately a directory walk + atomic update.